.classpath

```diff
@@ -43,6 +43,12 @@
 	<classpathentry kind="lib" path="lib/zap-2.5.0.jar"/>
 	<classpathentry kind="lib" path="lib/json-20160212.jar"/>
 	<classpathentry kind="lib" path="lib/commons-compress-1.13.jar"/>
+	<classpathentry kind="lib" path="lib/licence.jar"/>
+	<classpathentry kind="lib" path="lib/jxbrowser-6.12.jar"/>
+	<classpathentry kind="lib" path="lib/jxbrowser-linux32-6.12.jar"/>
```

Comment:
Agreed. I'll put them in zap-libs, but after https://github.com/zaproxy/zap-libs/pull/2 has been merged if thats ok?